src/pages/search/image.vue

```diff
@@ -9,19 +9,30 @@
 
 <script>
 import { propTypes } from '~/pages/search/search-page.types'
-import { computed, defineComponent } from '@nuxtjs/composition-api'
+import {
+  useStore,
+  computed,
+  defineComponent,
+  useMeta,
+} from '@nuxtjs/composition-api'
 import { useLoadMore } from '~/composables/use-load-more'
 
 const ImageSearch = defineComponent({
   name: 'ImageSearch',
   props: propTypes,
   setup(props) {
+    const store = useStore()
+
+    const query = computed(() => store.state.search.query.q)
+    useMeta({ title: `${query.value} - ${this.$t('hero.brand')}` })
```

Comment:
Oops, you can't use `this` in the setup function.

Reply:
Good catch, I'm confused as to why it works locally for me 😖.

Reply:
Updated in cc7a4f2.